client/web/compose/src/components/PageBlocks/base.vue

```diff
@@ -94,6 +94,12 @@ export default {
     autoRefreshEnabled () {
       return this.options.refreshRate >= 5 && ['page', 'page.record'].includes(this.$route.name)
     },
+
+    isModal () {
```

Comment:
Lets rename this to something that better describes it. 
isModal means the thing is a modal, which this is not. 
It should rather be something like `inModal` if you want to keep it short
A comment above describing it also helps